### ISSUE TITLE
Moving Libraries and Syncing

### ISSUE THREAD

**Issue by ghost** [HappenApps/Quiver#124]
Causes notebooks and notes to be deleted. Scenario: I can't really move a Quiver library from Documents to Box, for example using Quiver's move library function. The library remains in documents. So if I move it from  Documents to Box and click on the file directly in Finder, sometimes the entirety of my notes will be deleted.

Scenario 2: Backing up my library in multiple places doesn't actually back anything up. I opened a library file in my Box Sync folder, thinking I'd been backing up there the whole time, after Quiver somehow deleted my notes, and nothing, no notebooks, notes, trash. I still had an older copy in Dropbox but the notes I just created were missing.

So basically I can't move library files around in Finder without losing notes. I also can't backup to more than one Cloud service without notes being missing in one or the other.

I am going to uni for comp sci and I really wanted to use this as my primary notes app. Not so sure now.

**Comment by ylian** [comment]
That's odd. Some clarifications would help me.

Scenario 1: If you use Quiver's "move library" function, it simply moves the library file to the new location and reload the library. It should not delete any notebook or notes. Is this something you can reproduce?

Scenario 2: When you back up a Quiver library, it simply makes a copy of the library file, so it contains all the notebooks and notes at the moment of backup. A backup is not updated afterwards. Quiver only uses one library at a time, and a backup is a copy of the library at a certain time.

The best way to make sure all your notes are safe is to check the Quiver library into a Git repo.

**Comment by ghost** [comment]
I can try. And when I backup to Box there are no notes in that library file. I don't know why. But they never sync there.

**Comment by ylian** [comment]
Did you happen to choose the current library file when you try to back it up? That can trigger a bug that will delete your notes. I am aware of that bug and it will be fixed in the next update.

**Comment by ghost** [comment]
Yes, I think so.

**Comment by ghost** [comment]
Any eta on iOS and Quiver 2?

**Comment by ylian** [comment]
I see. Another user got tripped by the same nasty bug not so long ago. I will close this since that is fixed in the coming update.

**Comment by themoenen** [comment]
It seems like 8 years later this bug is still there... Just moved my library to Dropbox via the button in the app preferences, and years of content was deleted. I have an outdated version of the library on another computer, but I'm not able to restore it either (it still works on the other computer).

I can't recreate the problem it with a fresh library, which makes me conclude the problem is with the library itself. If you want I can email it to you, let me know.

Using latest 3.2.7 (201) on MacOS Sonoma 14.1.1 (23B81)
